cime_config/tests.py

```diff
@@ -427,42 +427,48 @@
     },
 
     "e3sm_scream" : {
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v0"),
+    },
+
+    "e3sm_scream_v0" : {
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v0_lowres"),
+    },
+
+    "e3sm_scream_v0_lowres" : {
         "time"  : "03:00:00",
         "tests" : (
             "SMS_D.ne4pg2_ne4pg2.F2010-SCREAM-HR",
             "SMS_D.ne4pg2_ne4pg2.F2010-SCREAM-LR",
-            "ERS.ne4pg2_ne4pg2.F2010-SCREAM-HR",
-            "ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR",
             "ERP.ne4pg2_ne4pg2.F2010-SCREAM-HR.eam-double_memleak_tol",
             "ERP.ne4pg2_ne4pg2.F2010-SCREAM-LR.eam-double_memleak_tol",
-            "REP.ne4pg2_ne4pg2.F2010-SCREAM-HR",
-            "REP.ne4pg2_ne4pg2.F2010-SCREAM-LR",
-            "PEM.ne4pg2_ne4pg2.F2010-SCREAM-HR",
-            "PEM.ne4pg2_ne4pg2.F2010-SCREAM-LR",
-            "SMS_R_Ln10.ne4_ne4.FDPSCREAMA97",
-            "ERS_R_Ln10.ne4_ne4.FDPSCREAMA97",
             "ERP_R_Ln10.ne4_ne4.FDPSCREAMA97",
             )
     },
 
     "e3sm_scream_v1" : {
-        "time"  : "03:00:00",
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v1_lowres", "e3sm_scream_v1_medres"),
+    },
+
+
+    "e3sm_scream_v1_lowres" : {
+        "time"  : "01:00:00",
         "tests" : (
-            "SMS_D_Ln2_P24x1.ne4_ne4.F2000SCREAMv1",
-            "SMS_D_Ln2_P24x1.ne4_ne4.F2010-SCREAMv1",
-            "SMS_D_Ln2_P24x1.ne4_ne4.F2000-SCREAMv1-AQP1",
+            "ERP_D_Ln9.ne4_ne4.F2010-SCREAMv1",
+            "ERS_D_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1",
```

Comment:
likewise, can we delete _D_ from one of these tests?

Reply:
Sure. Any preference of which one you want to run in production mode?

Reply:
Let's make the first one non-D: ERP_Ln9.ne4_ne4.F2010-SCREAMv1... that way we test aquaplanet opt build in this list, then aquaplanet debug build in the next.

Reply:
Wait, then you need the ERS to be non-D here.